dspace/config/registries/openaire4-relationships.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE relationships SYSTEM "relationships.dtd">
+
+<relationships>
+
+	<!-- https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/field_creator.html 
+		The relation between a Publication and a Person regarding the authorship -->
+	<type>
+		<leftType>Publication</leftType>
+		<rightType>Person</rightType>
+		<leftwardType>isAuthorOfPublication</leftwardType>
+		<rightwardType>isPublicationOfAuthor</rightwardType>
+		<leftCardinality>
+			<min>0</min>
+		</leftCardinality>
+		<rightCardinality>
+			<min>0</min>
+		</rightCardinality>
+	</type>
+	<!-- https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/field_creator.html 
+		The relation between a Publication and an Organization regarding the authorship -->
+	<type>
+		<leftType>Publication</leftType>
+		<rightType>Organization</rightType>
+		<leftwardType>isAuthorOfPublication</leftwardType>
+		<rightwardType>isPublicationOfAuthor</rightwardType>
```

Comment:
In other Entities examples, we've labeled these types `isOrgUnitofPublication` and `isPublicationOfOrgUnit`. 

@benbosman : Do duplicative labels here matter?  I notice we use `isAuthorOfPublication` and `isPublicationOfAuthor` as the labels for *both* the Publication <-> Organization AND Publication <-> Person relationships.   My assumption is this will be no big deal, but I wanted to double check as it jumped out to me.

Reply:
I think when we use `isOrgUnitofPublication` to describe a work we are referring to a belonging relation which is different from what this PR does. We are saying that an organization is related with a work in a sense it is a contributor or a creator of that work or publication.